.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=lf
```

Comment:
Forgot to ask, why is this needed?  Is it just for the tests?  If so, should it just live in in `tests` and target the `targets` and `fixtures` folders?

Reply:
Also, should we be added an editorconfig file(s) to specify our line ending policies?

Reply:
This is to make sure that the project is checked out with Linux EOL in source code and test (with exception of the *_CRLF folders) so `cargo fmt` won't messed up the files being different

Reply:
Oh right, the problem of aligning rustfmt with git.

This seems less obvious enough that it might deserve a comment in the file.